casper/src/main/scala/coop/rchain/casper/util/rholang/RuntimeManager.scala

```diff
@@ -79,270 +58,75 @@ trait RuntimeManager[F[_]] {
   def getContinuation(hash: StateHash)(
       channels: Seq[Par]
   ): F[Seq[(Seq[BindPattern], Par)]]
-  def emptyStateHash: StateHash
-  def withRuntimeLock[A](f: Runtime[F] => F[A]): F[A]
+  def emptyStateHash: F[StateHash]
```

Comment:
Ok. My idea is that this pr is only refactoring runtime but not `runtimeManager`. I didn't change traits on runtime manager on purpose. Actually, most of the methods in the runtimeManager trait is not really neccessary right now.

I want to refactor this runtimeManager on another pr. Ideally, my idea is that there would be only `withRuntime` and `withReplayRuntime` left in the manager and each `withRuntime` would spawn a new runtime and each runtime can be run in parallel which is what we discussed long before.

Reply:
But now to get `emptyStateHash` runtime must be taken from MVar which doesn't make sense because it's just the constant value, which also doesn't have to be wrapped in `F`. So it's not really a big change to move it to companion object for easy use.